pkg/minikube/bootstrapper/certs.go

```diff
@@ -545,3 +576,31 @@ func isValid(certPath, keyPath string) bool {
 
 	return true
 }
+
+func isKubeadmCertValid(cmd command.Runner, certPath string) bool {
```

Comment:
how about using linux to check if a cert is valid or not to avoid time zone difference between minikube and the host
```
Open a terminal window in Linux.

Type the following command, replacing <cert-file> with the path and name of the certificate file you want to check:

php
Copy code
openssl x509 -enddate -noout -in <cert-file>
Press Enter.

The output of the command will display the end date of the certificate in the format notAfter=YYYYMMDDHHMMSSZ.

For example:

makefile
Copy code
notAfter=20230410000000Z
Check the end date of the certificate. If the end date is in the past, then the certificate has expired.

For example, in the output above, the certificate will expire on April 10, 2023, at midnight UTC. If today's date is after April 10, 2023, then the certificate has expired.
```